bin/drop_filter_results.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+import argparse
+import re
+from collections import OrderedDict
+from pathlib import Path
+import csv
+import os
+import pandas as pd
+import pyreadr
+
+
+def FilteringResults(sample, gene_panel, out_drop_ae_rds, out_drop_gene_name, out_drop_as_tsv):
+    # Read gene panel if it has been provided
+    if gene_panel != "None":
+        df_panel = pd.read_csv(gene_panel, sep="\t", skiprows=26)
+
+    if out_drop_ae_rds != "None":
+        rds_AE = pyreadr.read_r(out_drop_ae_rds)
+        df_results_AE = rds_AE[None]
+        # Keep only samples provided to tomte
+        df_results_family_AE = df_results_AE.loc[df_results_AE["sampleID"].isin(sample)]
+        # Count how many events are significant per provided sample
+        df_family_AE_top20 = pd.DataFrame()
+
+        for id in sample:
+            df_id = df_results_family_AE[df_results_family_AE["sampleID"] == id]
+            if sum(df_id["padjust"] < 0.05) < 20:
```

Comment:
Perhaps we should have this as a parameter that one could optionally input as we give the user the possibility to adjust the adjusted p-value cut off when the config is generated. We can default it to 20 in the python script if nothing is given.

Reply:
I have solved it in another way :)
The user provides earlier the padjvalue and there is a column on the data frame that has aberrant true or false, I will filter on that instead